src/test/java/com/svydovets/bibirnate/cache/command/CacheConstant.java

```diff
@@ -0,0 +1,8 @@
+package com.svydovets.bibirnate.cache.command;
+
+public class CacheConstant {
```

Comment:
this looks like a constants class so can be made final and add private constructor